packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -569,6 +579,9 @@ export class ProjectService {
         exploreName: string,
         csvLimit: number | null | undefined,
     ): Promise<ApiQueryResults> {
+        if (!isUserWithOrg(user)) {
```

Comment:
considering how often you do this, maybe we could build a method called `expectUserWithOrg` or `assertUserWithOrg`